README.md

```diff
@@ -472,9 +472,9 @@ This table describes the list of available parameters for Helm Chart.
 | `bootstrapHostName`                 | Host name of MarkLogic bootstrap host                                                                | `""`   
 | `group.name`               | group name for joining MarkLogic cluster                                                                    | `Default`                              |
 | `group.enableXdqpSsl`                 | SSL encryption for XDQP                                                                   | `true`                         |
-| `licenseKey`                 | 	set MarkLogic license key installed                       | `""`
+| `license.key`                 | 	set MarkLogic license key installed                       | `""`
 | `licensee`                 | set MarkLogic licensee information                       | `""`
-| `installConverters`                 | Installs converters for the client if they are not already installed                       | `false`
+| `license.enableConverters`                 | Installs converters for the client if they are not already installed                       | `false`
```

Comment:
Looks like a typo here. Should be `enableConverters`. Love the `license.` prefix to the `licensee` above.